vocab/unit/VOCAB_QUDT-UNITS-ALL-v2.1.ttl

```diff
@@ -38480,10 +38483,11 @@ unit:PPT
   qudt:applicableSystem sou:SI ;
   qudt:conversionMultiplier 0.000000000001 ;
   qudt:conversionMultiplierSN 1.0E-12 ;
+  qudt:exactMatch unit:PPTR ;
   qudt:hasDimensionVector qkdv:A0E0L0I0M0H0T0D1 ;
   qudt:hasQuantityKind quantitykind:DimensionlessRatio ;
   qudt:iec61360Code "0112/2///62720#UAD927" ;
-  qudt:symbol "‰" ;
+  qudt:symbol "ppt" ;
```

Comment:
why this change?

Reply:
Because the symbol that had been there means "part per thousand" which we have as unit:PPTH. 
unit:PPT means "part per trillion".